server/routerlicious/packages/protocol-base/src/blobs.ts

```diff
@@ -42,14 +41,12 @@ export function getGitMode(value: SummaryObject): string {
  * @param value - summary object
  * @returns the type of summary object
  */
-export function getGitType(value: SummaryObject): string {
+export function getGitType(value: SummaryObject): "blob" | "commit" | "tree" | "attachment" {
     const type = value.type === SummaryType.Handle ? value.handleType : value.type;
 
     switch (type) {
         case SummaryType.Blob:
             return "blob";
-        case SummaryType.Commit:
```

Comment:
The function name `getGitType` and location in the code makes me feel like it should be possible to return "commit", since it is scoped to git functionality vs. the specific routerlicious usage of git objects.. if the latter is the case, then maybe at least a brief comment could clarify that this is scoped to routerlicious git objects only.

Reply:
Not sure I follow you. The return type is a function of input. I.e. if SummaryObject had commits, then getGitType would return commits. So it's more a function of summaries than git/r11s